docs/reference/pip_install.rst

```diff
@@ -89,6 +93,32 @@ Some Examples:
   Don't use single or double quotes in a ``requirements.txt`` file.
 
 
+.. _`Per-requirement Overrides`:
+
+Per-requirement Overrides
++++++++++++++++++++++++++
+
+When pip installs packages, it normally executes the ``setup.py`` file
+behind the scenes. You may extend the arguments with which the
+``setup.py`` file is called through the ``--global-option`` and
+``--install-option`` options. For example:
+
+ ::
+
+    FooProject >= 1.2 --global-option="--no-user-cfg" \
+                      --install-option="--prefix='/usr/local'" \
+                      --install-option="--no-compile"
+
+The above translates roughly into running FooProject's ``setup.py``
+script as:
+
+ ::
+
+   python setup.py --no-user-cfg install --prefix='/usr/local' --no-compile
+
+Note that the correct way of giving more than one option to
```

Comment:
the only way.   stuffing in multiple options results in an option error to setup.py